src/app/components/applications/SSV/MyAccount/components/EditFeeFlow/CancelUpdateFee/CancelUpdateFee.tsx

```diff
@@ -20,28 +19,33 @@ import { getStrategyRedirect } from '~app/redux/navigation.slice';
 import { getIsContractWallet } from '~app/redux/wallet.slice';
 import GoogleTagManager from '~lib/analytics/GoogleTag/GoogleTagManager';
 import { fromWei, getFeeForYear } from '~root/services/conversions.service';
+import { cancelChangeFeeProcess } from '~root/services/operatorContract.service.ts';
+import { fetchAndSetOperatorFeeInfo, getOperatorFeeData, getOperatorProcessId } from '~app/redux/operator.slice.ts';
 import { formatNumberToUi } from '~lib/utils/numbers';
 
 const CancelUpdateFee = () => {
   const stores = useStores();
   const classes = useStyles();
   const navigate = useNavigate();
-  const operatorStore: OperatorStore = stores.Operator;
   const processStore: ProcessStore = stores.Process;
   const process: SingleOperator = processStore.getProcess;
   const operator: IOperator = process.item;
   const [futureFee, setFutureFee] = useState(0);
   const [successPage, showSuccessPage] = useState(false);
+  const [isOpenCancelUpdateFeeDialog, setIsOpenCancelUpdateFeeDialog] = useState(false);
   const dispatch = useAppDispatch();
   const strategyRedirect = useAppSelector(getStrategyRedirect);
   const isContractWallet = useAppSelector(getIsContractWallet);
+  const operatorFeeData = useAppSelector(getOperatorFeeData);
+  const processOperatorId = useAppSelector(getOperatorProcessId);
 
   const cancelUpdateProcess = async () => {
-    if (!operatorStore.processOperatorId) return navigate(strategyRedirect);
-    const response = await operatorStore.cancelChangeFeeProcess({ operator, isContractWallet, dispatch });
+    if (!processOperatorId) return navigate(strategyRedirect);
+    const response = await cancelChangeFeeProcess({ operator, isContractWallet, dispatch });
     if (response) {
       // @ts-ignore
-      setFutureFee(operatorStore.operatorFutureFee);
+      await dispatch(fetchAndSetOperatorFeeInfo(operator.id));
+      setFutureFee(operatorFeeData.operatorFutureFee);
```

Comment:
make return in fetchAndSetOperatorFeeInfo to get response (operatorFeeData will be in object.payload)
don't use outer operatorFeeData from useSelect

Reply:
+